server/datastore/mysql/microsoft_mdm.go

```diff
@@ -960,11 +961,9 @@ SELECT
 FROM
     hosts h
     JOIN host_mdm hmdm ON h.id = hmdm.host_id
-    JOIN mobile_device_management_solutions mdms ON hmdm.mdm_id = mdms.id
+    JOIN mdm_windows_enrollments mwe ON h.uuid = mwe.host_uuid
 WHERE
-    mdms.name = '%s' AND
-    hmdm.is_server = 0 AND
```

Comment:
The new approach will still exclude any Windows server because we block them from enrolling in the first place, right?

Reply:
That's right, I will make a note just for our future selves

Reply:
I brought this back to be safe and because we're joining with `host_mdm` anyways.